tests/css-values-5.js

```diff
@@ -134,47 +134,410 @@ export default {
+				'mix(--scroll-progress-timeline by steps(10, start), left, right)',
+				'mix(50% of --keyframes-name, left, right)',
+			],
+		},
+		'mix() with value lists': {
+			links: {
+				tr: '#mix',
+				dev: '#mix',
+			},
+			properties: ['background-image'],
+			tests: [
+				'mix(\n  0.5;\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  50%;\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  --scroll-progress-timeline;\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  scroll();\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  view();\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  --scroll-progress-timeline by linear(0, 0.25, 1);\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  --scroll-progress-timeline by ease-in;\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  --scroll-progress-timeline by cubic-bezier(0.5, 0, 0.3, 1);\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  --scroll-progress-timeline by steps(10, start);\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+				'mix(\n  50% of --keyframes-name;\n  linear-gradient(red, transparent), radial-gradient(transparent, yellow);\n  linear-gradient(tramsüaremt. green), radial-gradient(blue, transparent)\n)',
+			],
+		},
+		'if() with lengths and percentages': {
+			links: {
+				tr: '#if-notation',
+				dev: '#if-notation',
+			},
+			tests: [
+				'if(media(width < 600px): 100px; else: 200px)',
+				'if(media(width < 600px): 10%; else: 20%)',
+				'if(supports(line-clamp: 3): 100px; else: 200px)',
+				'if(style(--responsive: true): 100px; else: 200px)',
+				'if(media(width < 600px): 100px; media(width > 1000px): 300px; else: 200px;)',
+				'if(media(width < 600px): 10%; media(width > 1000px): 30%; else: 20%)',
+				'if(supports(line-clamp: 3): 100px; supports(line-clamp: auto): 300px; else: 200px)',
+				'if(style(--responsive: true): 100px; style(--responsive: false): 300px; else: 200px)',
```

Comment:
Here are more test cases for reference only.
https://chromium-review.googlesource.com/c/chromium/src/+/6277288

Reply:
I've added more `if()` tests. Thank you for the hint, @yisibl!